src/components/form/button.ts

```diff
@@ -1,23 +1,25 @@
 import styled, { css } from 'styled-components'
-import { Paddings, Margins } from 'themes/function'
+import { Paddings, Margins, MarginsType, PaddingsType } from 'themes/function'
 import device from 'themes/device'
```

Comment:
do we need these imports -MarginsType, PaddingsType? I see  Paddings already mapped to PaddingsType and Margins mapped to MarginsType inside themes/function

Reply:
Otherwise I am getting this error -  

 Argument of type '({ m, mt, ml, mr, mb }: MarginsType) => FlattenSimpleInterpolation' is not assignable to parameter of type 'Interpolation<ThemedStyledProps<SharedButtonStyleProps, DefaultTheme>>'.
  Type '({ m, mt, ml, mr, mb }: MarginsType) => FlattenSimpleInterpolation' is not assignable to type 'InterpolationFunction<ThemedStyledProps<SharedButtonStyleProps, DefaultTheme>>'.
    Types of parameters '__0' and 'props' are incompatible.
      Type 'ThemedStyledProps<SharedButtonStyleProps, DefaultTheme>' has no properties in common with type 'MarginsType'.ts(2345)
      
For some reason the types are not recognized , please do suggest an alternative if any. @amina-deriv

Reply:
@prince-deriv